### DISCUSSION TITLE
import more than 4000 characters

### DISCUSSION THREAD

**Discussion by tuanitcm** [Justmalhar/open-audio#4] [Ideas]
Hello, is there any way to import more than 4000 characters? I find it really necessary.

**Comment by Justmalhar** [comment]
Hello @tuanitcm the OpenAI API currently only supports 4000 tokens maximum. I will update the repo if anything changes

**Reply by tuanitcm** [reply]
I saw that this website has a document upload mode. When I used it before, I could upload a file of 50,000 words and it still worked well. Can you update it for this tool https://ttsopenai.com/documents
